store/sqlstore/upgrade.go

```diff
@@ -312,8 +312,10 @@ func UpgradeDatabaseToVersion43(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion44(sqlStore SqlStore) {
-	// TODO: Uncomment following when version 4.4.0 is released
-	//if shouldPerformUpgrade(sqlStore, VERSION_4_3_0, VERSION_4_4_0) {
-	//	saveSchemaVersion(sqlStore, VERSION_4_4_0)
-	//}
+	if shouldPerformUpgrade(sqlStore, VERSION_4_3_0, VERSION_4_4_0) {
+		// Add the IsActive column to UserAccessToken.
+		sqlStore.CreateColumnIfNotExists("UserAccessTokens", "IsActive", "boolean", "boolean", "1")
+
+		saveSchemaVersion(sqlStore, VERSION_4_4_0)
```

Comment:
You should at least leave this line commented. Otherwise if someone else were to add something to this function, us devs would have some mild headaches.

I'm actually not sure if the `CreateColumnIfNotExists` is even necessary at all though. I think the column should be created automatically.

Reply:
I seem to remember getting an error until I added this, but I could be mistaken.

Reply:
While 4.4 is still in development the if statement and `saveSchemaVersion(sqlStore, VERSION_4_4_0)` bit should still be commented out but your create column line should still be run, like this https://github.com/mattermost/mattermost-server/commit/9a87bb3af68216b53ee8f89d6604c715c7b85b2d#diff-a788391365ece46cdf5c98badcdc3c32R261